kiwi/system/root_bind.py

```diff
@@ -88,9 +88,12 @@ def mount_kernel_file_systems(self):
         """
         try:
             for location in self.bind_locations:
-                if os.path.exists(location):
+                mount_point = os.path.normpath(os.sep.join([
```

Comment:
It turned out that some images did not have all the default kernel file systems folders (now I don't remember if it was CentOS or Fedora), thus is was breaking due to a non existing mountpoint.

I agree with the mountpoint name change. I am gonna change that.